bit/transaction.py

```diff
@@ -522,6 +515,65 @@ def construct_outputs(outputs):
     return outputs_obj
 
 
+def calculate_preimages(tx_obj, inputs_parameters):
+    # Transaction object used to create preimage must be contained in `tx_obj`
```

Comment:
Comments like this should probably be docstrings.

Reply:
I have tried to formulate a better docstring with commit 38d029f . Let me know if you have suggestions to improve the text.